resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/Client.scala

```diff
@@ -661,6 +665,33 @@ private[spark] class Client(
       }).toMap
     }).getOrElse(Map.empty[String, String])
   }
+
+  private def checkForUserJarsFilesExistence: Unit = {
+    val conf = new Configuration
+
+    def existsOrAbort(uri: URI): Unit = {
+      val path = new Path(uri)
+      if (!path.getFileSystem(conf).isFile(path)) {
```

Comment:
If we inline this in the `case` statement, it becomes clear that the scheme is of type `file`. Then instead of using `getFileSystem` we can just use an instance of `java.io.File` instance from the path component of the URI. This is the pattern we follow elsewhere, see `buildSubmissionRequest()`.